SVGKitLibrary/Demo-iOS/VCAllSpecImages.m

```diff
@@ -26,6 +27,10 @@ -(void)viewDidLoad
 		
 		self.svgFileNames = [NSMutableArray arrayWithArray: xcodeVirtualFolderSVGContents];
 	}
+    
```

Comment:
Quick check: is this new feature documented somewhere in the demo app?

It's important enough (makes a huge difference for what people are seeing) that I'd want it to be very obvious to users that you can do it.

Maybe instead we should have a segmented bar at the top or bottom of the page that lets you switch between the two modes at app-level, isntead of having to keep doing the prss-hold thing?

Reply:
(also because the new way you've implemented this, seems like it would be easier/better for the user interface to be switching this back and forth glboally)

Reply:
@adamgit > is this new feature documented somewhere in the demo app?

You mean long-press the collection cell and popup two menus `layer` or `fast` this feature ? It's added by this changes to see both two type of rendering result. Previouslly we only render all SVGs in the demo using `SVGFastImageView` (with only 3 hard-coded SVG here: ). However, it's not so convenient.

I think (or in my opinion), since we have two `SVGFastImageView` && `SVGLayeredImageview` representing two type of rendering engine, we'd better provide a way to check both of two. Adding a menu in the UI is the most easy way to do so...

If you means there are no UI tips or something to let user know we can long-press on the collection cell, I can add one tip, it's easy. 😅

Reply:
@adamgit The choice whether to use layer or fast is per-SVG level...And it's memorized until you long-press again...I can add a button, but however, there are already `Animate`, `Debug` two buttons on the navigation bar. It seems have no extra spaces...

Another issue is that current code about that detail pages does not support refresh or reload. If I want to support live-change from `SVGFastImageView` or `SVGLayeredImageView` I need to change too much current code (These code seems written long year ago 😅). So I think add a option in the previous page (The list page) is better...

Reply:
The choice whether to use `layer` or `fast` is per-SVG level...And it's memorized until you long-press again. I can add a button in the detail page, but however, there are already `Animate`, `Debug` two buttons on the navigation bar. It seems have no extra spaces...

Another issue is that current code about that detail pages does not support refresh or reload. If I want to support live-change from `SVGFastImageView` or `SVGLayeredImageView` I need to change too much current code (These code seems written long year ago 😅). So I finally adding a option in the previous page (The list page)...

Reply:
Anyway, if you don't mind I change too much of demo code, I can make the switch from `SVGFastImageView` && `SVGLayeredImageView` inside the detail page. Maybe I will remove the title, and adding a new navigation bar button.

If the button is showing `layer`, it's now rendered as `SVGLayeredImageView`, click it the button then the current page will reloaded with `SVGFastImageView`.
If the button is showing `fast`, it's now rendered as `SVGFastImageView`, click the button then the current page will reloaded with `SVGLayeredImageView`.